pkg/jx/cmd/create_addon_vault.go

```diff
@@ -77,11 +77,12 @@ func NewCmdCreateAddonVault(f Factory, in terminal.FileReader, out terminal.File
 
 // Run implements the command
 func (o *CreateAddonVaultOptions) Run() error {
-	_, _, err := o.KubeClient()
```

Comment:
This call to `o.KubeClient()` seemed to be a bit pointless

Reply:
This is just to check if there is a kuberntes connection when running the create addon. It can be ignored.